package.json

```diff
@@ -21,9 +21,11 @@
 		"@wordpress/blob": "file:packages/blob",
 		"@wordpress/block-serialization-spec-parser": "file:packages/block-serialization-spec-parser",
 		"@wordpress/blocks": "file:packages/blocks",
+		"@wordpress/code-editor": "file:packages/code-editor",
 		"@wordpress/components": "file:packages/components",
 		"@wordpress/compose": "file:packages/compose",
 		"@wordpress/core-data": "file:packages/core-data",
+		"@wordpress/core-editor": "file:packages/code-editor",
```

Comment:
Is this line intentional? `core-editor` , not `code-editor` ?

Reply:
definitely not :)